src/home/ExpandedCard.js

```diff
@@ -4,13 +4,34 @@ import {
   Text,
   ScrollView,
   Dimensions,
+  Image,
 } from 'react-native';
-import AddButton from './components/AddToLessonButton';
+import { useState, useEffect } from 'react';
+import { readFile } from '../services/routes/Local.js';
 import Header from '../Components/Header';
 
 const windowHeight = Dimensions.get('window').height;
 
-const ExpandedCard = ({ navigation }) => {
+const ExpandedCard = ({ route, navigation }) => {
+  const { cardPath } = route.params;
+  const [title, setTitle] = useState('');
+  const [cardImagePath, setCardImagePath] = useState(null);
+
+  useEffect(() => {
+    const readCardTitle = async () => {
+      try {
+        const cardTitlePath = cardPath + 'cardName.txt';
+        var cardNames = await readFile(cardTitlePath, 'utf-8');
```

Comment:
```suggestion
        let cardNames = await readFile(cardTitlePath, 'utf-8');
```
Very small thing but `let` is generally better practice because `var` is globally scoped